### PR TITLE
Document ILM waits for tsds end_time to lapse in some actions

### PR DESCRIPTION
**Pull request by andreidan**

### CODE REVIEW

File: docs/reference/data-streams/downsampling-ilm.asciidoc [hunk 1]
```diff
@@ -37,8 +37,16 @@ To enable downsampling, add a <<ilm-downsample,Downsample action>> and set
...
+takes place after the index is rolled over and the <<index-time-series-end-time, index time series end time>>
+has lapsed as the source index is still expected to receive writes until then.
+{ilm-cap} will not proceed with any action that expects the index to not receive
+writes anymore until the <<index-time-series-end-time, index's end time>> has 
+passed. The {ilm-cap} actions that wait on the end time before proceeding are:
+- <<ilm-downsample>>
+- <<ilm-forcemerge>>
+- <<ilm-readonly>>
+- <<ilm-searchable-snapshot>>
+- <<ilm-shrink>>
```
Comment (martijnvg) [inline docs/reference/data-streams/downsampling-ilm.asciidoc:49]: and ilm-delete? (when other pr is merged)

File: docs/reference/data-streams/downsampling-ilm.asciidoc [hunk 2]
```diff
@@ -37,8 +37,16 @@ To enable downsampling, add a <<ilm-downsample,Downsample action>> and set
 which you want to aggregate the original time series data.
 
 In this example, an ILM policy is configured for the `hot` phase. The downsample
-takes place after the initial index rollover, which for demonstration
-purposes is set to run after five minutes.
+takes place after the index is rolled over and the <<index-time-series-end-time, index time series end time>>
+has lapsed as the source index is still expected to receive writes until then.
+{ilm-cap} will not proceed with any action that expects the index to not receive
```
Comment (martijnvg) [inline docs/reference/data-streams/downsampling-ilm.asciidoc:42]: receive major writes? After end_time is passed a backing index can still accept writes, but the expectation is that the volume of writes will be lower.

**Review by martijnvg — APPROVED** [review]
LGTM

File: docs/reference/data-streams/downsampling-ilm.asciidoc (hunk 1 above)
Comment (andreidan) [reply docs/reference/data-streams/downsampling-ilm.asciidoc:50]: ++ done, thanks Martijn

**Pull request merged**